README.rst

```diff
@@ -1175,7 +1175,7 @@ Choose from the list of available rules:
   There should not be an empty ``return`` statement at the end of a
   function.
 
-* **no_whitespace_before_comma_in_array** [@Symfony, @PhpCsFixer]
+* **no_whitespace_before_comma_in_array** [@Symfony, @PhpCsFixer, @PHP73Migration]
```

Comment:
how is it related to 7.3 ?

Reply:
the `after_heredoc` option is desired while migrating to php 7.3. That is the reason, why I've added the fixer with that option to the ruleset.